src/adapter/search-response-adapter/__tests__/pagination-adapter.tests.ts

```diff
@@ -42,85 +47,133 @@ const paginateHitsTestsParameters = [
     hits: [],
     page: 0,
     hitsPerPage: 20,
-    returnedHits: [],
+    returnedPagination: {
+      page: 0,
+      hitsPerPage: 20,
+      nbPages: 0,
+    },
   },
   {
     hits: [],
     page: 100,
     hitsPerPage: 0,
-    returnedHits: [],
+    returnedPagination: {
+      page: 100,
+      hitsPerPage: 0,
+      nbPages: 0,
+    },
   },
   {
     hits: [],
     page: 100,
     hitsPerPage: 20,
-    returnedHits: [],
+    returnedPagination: {
+      page: 100,
+      hitsPerPage: 20,
+      nbPages: 0,
+    },
   },
 
   // Page 0
   {
     hits: [{ id: 1 }, { id: 2 }, { id: 3 }],
     page: 0,
     hitsPerPage: 20,
-    returnedHits: [{ id: 1 }, { id: 2 }, { id: 3 }],
+    returnedPagination: { page: 0, hitsPerPage: 20, nbPages: 1 },
```

Comment:
I think you can opt to use multi-line or one-line but as soon you opt by one version you must keep the consistency :)